### PR TITLE
Update Godot versions

### PR DESCRIPTION
**Pull request by ignaloidas**
If you're updating an existing documentation to it's latest version, please ensure that you have:

- [x] Updated the versions and releases in the scraper file
- [x] Ensured the license is up-to-date and that the documentation's entry in the array in `about_tmpl.coffee` matches it's data in `self.attribution`
- [x] Ensured the icons and the `SOURCE` file in <code>public/icons/*your_scraper_name*/</code> are up-to-date if the documentation has a custom icon
- [x] Ensured `self.links` contains up-to-date urls if `self.links` is defined
- [ ] Tested the changes locally to ensure:
  - The scraper still works without errors
  - The scraped documentation still looks consistent with the rest of DevDocs
  - The categorization of entries is still good

### CODE REVIEW

**Comment by ignaloidas** [comment]
Unable to test the changes locally currently, but looks like everything should be fine.

**Comment by j-f1** [comment]
It looks like the 3.0 series is no longer available from https://godotengine.org/download. Should we still host the docs?

**Comment by ignaloidas** [comment]
After some discussion in Godot's IRC 3.0 and 3.1 versions are dropped as 3.0 is not really used anymore and 3.1 is compatible with 3.2 docs

**Comment by allanjoh** [comment]
I was not able to get this running locally: 
```
 bundle exec thor docs:generate godot --version=3.2
/!\ WARNING /!\

Some scrapers send thousands of HTTP requests in a short period of time,
which can slow down the source site and trouble its maintainers.

Please scrape responsibly. Don't do it unless you're modifying the code.

To download the latest tested version of this documentation, run:
  thor docs:download godot

Proceed? (y/n) y
[###################################################################################################] [1/1] [100.00%] [00:00] [00:00] [1.26/s]Failed! (try running with --debug for more information)
```

However, if we change the protocol to `https` then it succeeds

File: lib/docs/scrapers/godot.rb [hunk 1]
```diff
@@ -28,18 +28,13 @@ class Godot < UrlScraper
...
 
-    version '3.1' do
-      self.release = '3.1.1'
-      self.base_url = "http://docs.godotengine.org/en/#{self.version}/"
-    end
-
-    version '3.0' do
-      self.release = '3.0.6'
+    version '3.2' do
+      self.release = '3.2.1'
```
Comment (Calinou) [inline lib/docs/scrapers/godot.rb:32]: Godot 3.2.3 is the latest stable version now:

```suggestion
      self.release = '3.2.3'
```

**Review by simon04 — APPROVED** [review]

**Pull request merged**